crowdsourcer/management/commands/import_foi_data.py

```diff
@@ -0,0 +1,323 @@
+            "question": 6,
+            "type": "yes_no",
+        },
+        "C&E Q3": {
+            "section": "Buildings & Heating",
+            "question": 8,
+            "type": "yes_no",
+        },
+        "G&F Q8": {
+            "section": "Governance & Finance",
+            "question": 8,
+            "type": "yes_no",
+        },
+        "G&F Q9": {
+            "section": "Governance & Finance",
+            "question": 9,
+            "type": "yes_no",
+        },
+        "Planning 11": {
+            "section": "Planning & Land Use",
+            "question": 11,
+            "type": "yes_no",
+        },
+        "Transport 11ii": {
+            "section": "Transport",
+            "question": 11,
+            "type": "yes_no",
+        },
+    }
+
+    tiered_options = {
+        "B&H Q2": {
+            "options": [
+                {"description": "100% green energy", "score": 1},
+                {"description": "Green Tariff", "score": 1},
+                {"description": "Creates 20% own energy", "score": 1},
+                {"description": "Criteria not met", "score": 0},
+                {"description": "No response", "score": 0},
+            ]
+        },
+        "B&H Q3": {
+            "options": [
+                {"description": "50% or above", "score": 1},
+                {"description": "60% or above", "score": 2},
+                {"description": "90% or above", "score": 3},
+                {"description": "Criteria not met", "score": 0},
+                {"description": "No response", "score": 0},
+            ]
+        },
+    }
+
+    authority_name_map = {
+        "Lincoln City Council": "City of Lincoln",
+        "King's Lynn and West Norfolk Borough Council": "Kings Lynn and West Norfolk Borough Council",
+        "Common Council of the City of London": "City of London",
+        "London Borough of Barking and Dagenham Council": "Barking and Dagenham Borough Council",
+    }
+
+    def get_council_lookup(self):
+        url = get_dataset_url(
+            repo_name="uk_local_authority_names_and_codes",
+            package_name="uk_la_future",
+            version_name="1",
+            file_name="uk_local_authorities_future.csv",
+            done_survey=True,
+        )
+        df = pd.read_csv(url)
+
+        lookup = {}
+        for _, row in df.iterrows():
+            gss = row["gss-code"]
+            lookup[row["nice-name"].replace(" ", "-").lower()] = gss
+            lookup[row["nice-name"]] = gss
+            lookup[row["official-name"]] = gss
+
+        return lookup
+
+    def get_option_for_question(self, q, option):
+        option = Option.objects.get(question=q, description=option)
+
+        return option
+
+    def get_bh_3_answer(self, q, row, value):
+        if value == 0:
+            description = "Criteria not met"
+        elif value == 1:
+            try:
+                percentage = row.iloc(8)
+            except ValueError:
+                percentage = 50
```

Comment:
This is fine *if* value = 0 but there are no extra details *always* means 50. 

Along similar grounds, possibly should assert here that percentage is never below 50 (in case the first option is true, but they've messed up here).

Reply:
This is mostly to handle the case where they have selected 1 and then not filled in the percentages so I've assigned the minimum score.